### PR TITLE
Update requestsent

### PR DESCRIPTION
**Pull request by ifndefdeadmau5**
To update userlist when request sent, use 'Actions.main' with props 'me' at RequestSent.js

### CODE REVIEW

File: src/components/userProfile/RequestPage.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ class RequestPage extends Component {
     if (error) {
```
Comment (MechanicKim) [inline src/components/userProfile/RequestPage.js:68]: Is it possible that error and result are null each other?

Comment (ifndefdeadmau5) [reply src/components/userProfile/RequestPage.js:68]: Hmm.. sorry! I have no idea about this. I just add parameter in 'Action.requestSent' method call.

**Comment by MechanicKim** [comment]
@ifndefdeadmau5

**Comment by MechanicKim** [comment]
After updating this branch, we can pass this PR! ^^

**Comment by MechanicKim** [comment]
No comment! LGTM ^^

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/118/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**